padertorch/contrib/examples/source_separation/or_pit/README.md

```diff
@@ -19,13 +19,13 @@ The training procedure of the OR-PIT consists of two steps: no fine-tuning and f
 The training for the first step can be initialized with:
      
 ```bash
-$ python -m padertorch.contrib.examples.or_pit.train init with database_jsons=${path_to_your_jsons}
+$ python -m padertorch.contrib.examples.source_separation.or_pit.train init with database_jsons=${path_to_your_jsons}
  ```
```

Comment:
Shouldn't we advice the user to directly start the training without using init?

Reply:
Yes, I agree, we should do that

Reply:
Done